add column cs_user_permissions for table alicloud_ram_user. Closes #181 (#183)
turbot/steampipe-plugin-alicloud: docs/tables/alicloud_ram_user.md

where
  not mfa_enabled;
```

> ### List all the users for whom role-based access control (RBAC) authorization is Enabled on Kubernetes Engine Clusters

Reviewer: ```suggestion
### List users with Container Service for Kubernetes role-based access control (RBAC) permissions
```
